pkg/cmd/config.go

```diff
@@ -522,6 +522,10 @@ systemPackages:
   # will add Docker and Kubernetes repositories to OS package manager
   configureRepositories: true # it's true by default
 
+addons:
+  enable: false
+  path: "./addons"
```

Comment:
Please mention here the fact that this path will be relative (in case when it's relative) to the config file itself.